### DISCUSSION TITLE
CleanFeed Custom Hide

### DISCUSSION THREAD

**Discussion by karhidish** [AprilSylph/XKit-Rewritten#355] [Suggestion Box]
Hi! I'm new here so apologies if I say anything against custom or if I don't understand how things work. 

I'm interested in whether it's possible to give tumblr users the ability to customize a list of blogs they can hide media from. The current options are "flagged blogs" and "all blogs," which are useful but not customizable. The hide avatar function, on the other hand allows users to create a list of usernames of blogs to hide avatars of. 

Would it be feasible to implement the ability to create a similar list as part of the already existing CleanFeed function, or would it take a lot of complex work?

**Comment by AprilSylph** [comment]
I think it would be feasible! I'll create an issue for this so that it's more visible.

**Reply by karhidish** [reply]
Thank you! :)
